test/FastTests/Issues/RavenDB_16590.cs

```diff
@@ -69,7 +69,8 @@ public void StudioListsOfIndexScopedSettings()
                 "Indexing.NumberOfLargeSegmentsToMergeInSingleBatch",
                 "Indexing.ScratchSpaceLimitInMb",
                 "Indexing.Throttling.TimeIntervalInMs",
-                "Indexing.TransactionSizeLimitInMb"
+                "Indexing.TransactionSizeLimitInMb",
+                "Indexing.Static.SearchEngineType"
```

Comment:
Also need to add `Indexing.Auto.SearchEngineType` ?

Reply:
No, auto is per database but here we have:
`RavenConfiguration.AllConfigurationEntries.Value
                    .Where(x => x.Scope == ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)`